src/network-config.js

```diff
@@ -32,17 +37,20 @@ export const networkConfigs = {
       portisDappId ? { id: 'portis', conf: portisDappId } : null,
     ].filter(p => p),
   },
```

Comment:
I thought the above renamings were more clear:

- `endpoints` communicates that this is a network-specific URL much better than `nodes.defaultEth`
- `environment` communicates that this is what will be available in the `environment.js` file much better than `settings` does (it's not really network settings anymore)